app/views/items/_item_list.html.erb

```diff
@@ -8,14 +8,14 @@
     <tr>
       <th>Category</th>
       <th>Name</th>
-      <th>Quantity Per Individual</th>
-      <th>Fair Market Value (per item)</th>
+      <th class = "text-right">Quantity Per Individual</th>
```

Comment:
I agree that Quantity Per Individual *should* be aligned right,  but the information in the column isn't,  so it looks weird.    It'd be fine if you wanted to go into the _item_row.html.erb and make the item_row.distribution_quantity aligned right as well.